support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -135,8 +148,20 @@ export function OneTimeCheckout({ geoId, appConfig }: OneTimeCheckoutProps) {
 
 	const stripePromise = loadStripe(stripePublicKey);
 
+	const elementsOptions = {
+		mode: 'payment',
+		/**
+		 * Stripe amounts are in the "smallest currency unit"
+		 * @see https://docs.stripe.com/api/charges/object
+		 * @see https://docs.stripe.com/currencies#zero-decimal
+		 */
+		amount: 100,
```

Comment:
If the user doesn't change their selection will they be charged £1? Maybe can load the default amount here or run one `elements.?update` as soon as the final amount is ready in the component

Reply:
Agree, its now been set the the minimum at start (but it does immediately use the config default one-time amount). Should finalAmount be undefined (our preferred amount over NaN/0) the ExpressCheckout is disabled in the useEffect (comment update there too). Thanks